packages/protocol/test/governance/voting/lockedgold.ts

```diff
@@ -63,7 +63,7 @@ contract('LockedGold', (accounts: string[]) => {
   beforeEach(async () => {
     mockGoldToken = await MockGoldToken.new()
     accountsInstance = await Accounts.new()
-    lockedGold = await LockedGold.new()
+    lockedGold = await LockedGold.new(true)
```

Comment:
we should add tests for reverting on `lock`, `relock`, and `unlock` when `value == 0`

Reply:
@yorhodes I have added the tests for the above scenario.